javascript-primitives-and-variables/.eslintrc

```diff
@@ -0,0 +1,76 @@
+{
+  "root": true,
+  "env": {
+    "node": true,
+    "browser": true
+  },
+  "parser": "babel-eslint",
+  "extends": [
+    "plugin:react/recommended",
+    "eslint:recommended",
+    "standard"
+  ],
+  "rules": {
+    "semi": [
+      "error",
+      "always"
+    ],
+    "padded-blocks": 0,
+    "arrow-parens": [
+      "error",
+      "as-needed"
+    ],
+    "curly": [
+      "error",
+      "multi-line"
+    ],
+    "no-console": [
+      "error",
+      {
+        "allow": [
+          "warn",
+          "error"
+        ]
+      }
+    ],
+    "space-before-function-paren": [
+      "error",
+      {
+        "anonymous": "always",
+        "named": "never",
+        "asyncArrow": "always"
+      }
+    ]
+  },
+  "overrides": [
+    {
+      "files": [
+        "*.test.js"
+      ],
+      "env": {
+        "mocha": true
+      }
+    },
+    {
+      "files": [
+        "**/*.jsx"
+      ],
+      "settings": {
+        "react": {
+          "version": "detect"
+        }
+      },
+      "rules": {
+        "react/prop-types": 0
+      }
+    },
+    {
+      "files": [
+        "**/*.html"
+      ],
+      "plugins": [
+        "eslint-plugin-html"
+      ]
+    }
+  ]
+}
```

Comment:
This should have been changed in the main `c0121-code-solutions` folder, not in this exercise folder. Please make that change and commit it, it will be important for future exercises.

Reply:
Hi Cody, i've gone ahead and done that. Thank you for the heads up!